jangjuseong/controllers/postController.js

```diff
@@ -0,0 +1,48 @@
+const postService = require('../services/postService');
+
+const createPost = async (req, res) => {
+  const { title, content, postImgUrl, userId } = req.body;
+
+  await postService.postUp(title, content, postImgUrl, userId);
+
+  return res.status(200).json({ message: 'postCreated' });
+};
+
+const getPost = async (req, res) => {
+  const result = await postService.postGet();
+
+  return res.status(200).json({ data: result });
+};
+
+const getPostbyUser = async (req, res) => {
```

Comment:
userId로 특정 게시글을 조회하고 있는 함수로 보이네요!
함수명을 아래와 같이 수정해주면 좋겠습니다.

```suggestion
const getPostbyUserId = async (req, res) => {
```